.gitignore

```diff
@@ -14,5 +14,8 @@ bin
 .ignored_out
 dist
 *.port
-**/unit-test-results
+**/*test-results
```

Comment:
I'm absolutely fine with this change but how do test results actually end up in the `test-results` folder within e2e-tests? Seems like I only get the `playwright-report`.